settings.gradle.kts

```diff
@@ -1,7 +1,7 @@
 buildscript { repositories { gradlePluginPortal() } }
 
 plugins {
-    id("net.vivin.gradle-semantic-build-versioning") version "4.0.0"
+    id("net.vivin.gradle-semantic-build-versioning") version "3.0.0"
```

Comment:
I see this is being removed in a different PR, but should it not change in this one?